src/backend/commands/explain.c

```diff
@@ -80,11 +75,7 @@ static void ExplainOneQuery(Query *query, int cursorOptions,
 static void report_triggers(ResultRelInfo *rInfo, bool show_relname,
 							ExplainState *es);
 
-#ifdef USE_ORCA
-static void ExplainDXL(Query *query, ExplainState *es,
-							const char *queryString,
-							ParamListInfo params);
-#endif
+
```

Comment:
I suggest removing the empty lines between `report_triggers` and `elapsed_time` to achieve more similarity between GPDB and PostgreSQL

https://github.com/postgres/postgres/blob/15afb7d61c142a9254a6612c6774aff4f358fb69/src/backend/commands/explain.c#L79-L81